#UAV Reference Commands
#Contains desired quantities
Header header
uint8 mode
geometry_msgs/Vector3 euler
float64 altitude
float64 airspeed
float64 groundspeed
